vnext/local-cli/generator-windows/templates/cpp/proj-experimental/MyApp.vcxproj

```diff
@@ -151,11 +145,20 @@
       <DeploymentContent>false</DeploymentContent>
     </Text>
   </ItemGroup>
-
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
-  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\Bundle.props"/>
-  <Import Project="$(ReactNativeWindowsDir)\PropertySheets\Bundle.Cpp.targets"/>
-
+  <PropertyGroup>
+    <UseExperimentalNuget>true</UseExperimentalNuget> <!-- This will eventually default to true and this property group should be removed -->
+  </PropertyGroup>
+  <ImportGroup Label="ReactNativeWindowsNodeTargets">
+    <Import Project="..\..\node_modules\react-native-windows\PropertySheets\External\Microsoft.ReactNative.CppApp.targets" Condition="Exists('..\..\node_modules\react-native-windows\PropertySheets\External\Microsoft.ReactNative.CppApp.targets')" />
```

Comment:
We can't assume this path.  The logic earlier in the file for ReactNativeWindowsDir that you deleted is what node guarantees.   -- I'll point out that the path you hard coded here wouldn't work for the react-native-windows repo.  Or the fluentui repo.  Or probably a bunch of other repos that contain more than one project.

Reply:
does the cli need to support mono repos out of the box? or it that something that a mono-repo has to customize for their stuff anyway